src/sdk-reference/js/6/offline-tools/index.md

```diff
@@ -7,69 +7,91 @@ order: 400
 
 # Offline tools
 
-The Kuzzle SDK provides a set of properties that help your application to be resilient to the loss of network connection
+The Kuzzle SDK provides a set of properties that helps your application to be resilient to the loss of network connection
```

Comment:
```suggestion
The Kuzzle SDK provides a set of properties that help your application to be resilient to the loss of network connection
```

Reply:
I disagree, it needs a 's' since it's "a set" that helps